docs/community/contribution.md

```diff
@@ -7,6 +7,24 @@ As a contributor, here are the guidelines we would like you to follow.
 ---
 Please read and follow our [Code of Conduct][coc].
 
+## Getting started
+---
+Get the Ergonode Source Code:
+* Create a [GitHub][github-join] account and sign in
+* Click on the "Fork button" in one of [Ergonode][github] repositories:
+  * [backend][github-backend]
+  * [frontend][github-frontend]
+  * [docker][github-docker]
+  * [docs][github-docs]
+* After the “forking action” has completed, clone your fork locally (for example use [docs][github-docs])
+```bash
+git clone git@github.com:YOUR_USERNAME/docs.git
+```
+* Add the upstream repository as a remote
+```bash
+cd docs
+git remote add upstream git@github.com:ergonode/docs.git
```

Comment:
Please add info on how to set up PR and also how to maintain origin branches - merges from upstream etc